core/controllers/base.py

```diff
@@ -357,12 +357,9 @@ def render_template(self, filepath, iframe_restriction='DENY'):
         self.response.headers['X-Content-Type-Options'] = 'nosniff'
         self.response.headers['X-Xss-Protection'] = '1; mode=block'
 
-        if iframe_restriction is not None:
-            if iframe_restriction in ['SAMEORIGIN', 'DENY']:
-                self.response.headers['X-Frame-Options'] = iframe_restriction
-            else:
-                raise Exception(
-                    'Invalid X-Frame-Options: %s' % iframe_restriction)
+        assert iframe_restriction in ['SAMEORIGIN', 'DENY'], (
```

Comment:
We don't use assert for error checking. Please raise an exception instead. If you'd like to make this shorter, consider placing this into a function.

Reply:
Done